src/app-bar.jsx

```diff
@@ -314,26 +313,26 @@ const AppBar = React.createClass({
           {children}
       </Paper>
     );
-  },
+  }
 
+  @autobind
```

Comment:
Instead of the arrow functions I could alternatively just leave `react-codemod` alone; it automatically puts `this` bindings in the constructor. Which would you prefer?